independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/Dockerfile-layout.include.qute

```diff
@@ -83,6 +83,10 @@ ENV LANGUAGE='en_US:en'
 
 {#insert copy /}
 
+# Remote Development Mode
+ENV QUARKUS_LAUNCH_DEVMODE=true
+RUN chmod o+rw -R /deployments
+
```

Comment:
Is it really something we want to include in the default Dockerfile?

I could be convinced to provide a specific remote dev mode Dockerfile if it has value but the default one should be as generic as possible and allowing the app to write in `/deployment` is not something we want if not in remote dev mode.

One option could be to comment it out but given they are very different use cases and you might need to use remote dev mode and then to build a container and it could be dangerous if you forgot to adjust it before building the prod container.

Reply:
Yeah so for me either we do doc only or we provide a separate Dockerfile. I'm not against the latter if you think it has value. Especially since if you are using remote dev, there's a good chance you will want two separate ones for the reason below.

> we could add a precision in the comment (do not leave uncommented when building the prod container)

This is too dangerous. When you have to add this sort of comment, you can be sure it will go wrong for someone somewhere :).
Or even get pushed to Git because you were working on a large patch and it went unnoticed.

Reply:
I think doc only is sufficient. I will make the change. thanks for the feedback.

Reply:
I have also added `For security reasons, this option should not be added to the production docker file.`  in the doc.
and reverted the other changes.